src/teleop/gui/src/components/JoystickValues.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+<div class = "datagrid">
+    <div>
+        <p style="margin-top:0px;">left_right: {{joystick_values.left_right}}</p>
+        <p>forward_back: {{joystick_values.forward_back}}</p>
+        <p>twist: {{joystick_values.twist}}</p>
+    </div>
+    <div>
+        <p style="margin-top:0px;">dampen: {{joystick_values.dampen}}</p>
+        <p>pan: {{joystick_values.pan}}</p>
+        <p>tilt: {{joystick_values.tilt}}</p>
```

Comment:
Is this for when we enter teleop mode?

Reply:
Can you also just add a screenshot of this part to the PR?

Reply:
I think we need to merge with master, this is a change from a PR last week or so.